sdk-commons/src/main/java/org/openmrs/maven/plugins/model/BaseSdkProperties.java

```diff
@@ -171,6 +172,48 @@ public List<ContentPackage> getContentPackages() {
 		return contentPackages;
 	}
 
+    /**
+     * Get all artifact-specific GitHub Packages configurations defined in the properties
+     * @return List of {@link GithubPackage} configurations
+     */
+    public List<GithubPackage> getGithubPackage() {
+        Map<String, GithubPackage> packages = new HashMap<>();
+        for (String key : properties.stringPropertyNames()) {
+            if (key.contains(".github.packages.")) {
+                String[] parts = key.split("\\.github\\.packages\\.");
+                if (parts.length != 2) continue;
+
+                String artifactKey = parts[0];
+                String configKey = parts[1];
+                GithubPackage pkg = packages.computeIfAbsent(artifactKey, k -> {
+                    GithubPackage newPkg = new GithubPackage();
+                    newPkg.setArtifactKey(k);
+                    return newPkg;
+                });
+
+                String value = properties.getProperty(key);
+                switch (configKey) {
+                    case "username":
+                        pkg.setUsername(value);
+                    break;
+                    case "token":
+                        pkg.setToken(value);
+                        break;
```

Comment:
I wouldn't think we would want to encourage / support credentials (username and/or token) being added directly to the distro properties file.  These would seem to be something that should come from some an environment variable or some other maven variable (eg. taken from settings.xml).